quiz.js

```diff
@@ -0,0 +1,44 @@
+var grow= [];
+var branchNum= document.getElementById("branches");
+var leafNum= document.getElementById("leaf");
+var button = document.getElementById("button");
+
+function tree(branches, leaf) {
```

Comment:
As per the instructions, this function should accept a single object containing keys for the height and characer as its input.